File: NHSOnline.Backend.Worker.IntegrationTests/Mocking/Emis/Models/DemographicsResponse.cs
using System;
using System.Collections.Generic;

namespace NHSOnline.Backend.Worker.IntegrationTests.Mocking.Emis.Models
{
    public class DemographicsResponse
    {
        public string Title { get; set; }
        public string FirstName { get; set; }
        public string Surname { get; set; }
        public string CallingName { get; set; }
        public IList<PatientIdentifier> PatientIdentifiers { get; set; }
        public DateTime DateOfBirth { get; set; }
        public Sex Sex { get; set; }
        public ContactDetails ContactDetails { get; set; }
        public Address Address { get; set; }
    }
}

File: src/main/kotlin/mocking/emis/models/DemographicsResponse.kt
package mocking.emis.models

import java.time.LocalDateTime

data class DemographicsResponse(
        var title: String? = null,
        var firstName: String? = null,
        var surname: String? = null,
        var callingName: String? = null,
        var patientIdentifiers: MutableList<PatientIdentifier>? = null,
        var dateOfBirth: LocalDateTime? = LocalDateTime.MIN,
        var sex: Sex? = null,
        var contactDetails: ContactDetails? = null,
        var address: Address? = null
)